app/routes/_layout+/bookings.new.tsx

```diff
@@ -154,17 +157,23 @@ export async function action({ context, request }: ActionFunctionArgs) {
       senderId: authSession.userId,
     });
 
-    const manageAssetsUrl = `/bookings/${
-      booking.id
-    }/add-assets?${new URLSearchParams({
-      // We force the as Date because we know that the booking.from and booking.to are set and exist at this point.
-      bookingFrom: (booking.from as Date).toISOString(),
-      bookingTo: (booking.to as Date).toISOString(),
-      hideUnavailable: "true",
-      unhideAssetsBookigIds: booking.id,
-    })}`;
-
-    return redirect(manageAssetsUrl);
+    const url = new URL(request.url);
+    const assetIds = url.searchParams.get("assetIds");
+    if (assetIds) {
```

Comment:
Instead of checking it like this, you should create a clear boolean so the code is better to read.

Reply:
Created a clear boolean. Resolved!